src/framework/components/camera/post-effect-queue.js

```diff
@@ -353,6 +349,7 @@ class PostEffectQueue {
         if (this.resizeTimeout)
             return;
 
+        // Note: this should be reviewed, as this would make postprocessing incorrect for few frames till reize takes place
```

Comment:
```suggestion
        // Note: this should be reviewed, as this would make postprocessing incorrect for a few frames 
        // until the resize takes place
```